copy-artifacts/.gitignore

```diff
@@ -0,0 +1 @@
+Cargo.lock
```

Comment:
copy-artifacts is now part of our main workspace, so it doesn't use a separate Cargo.lock file, but one was present